lib/zig/builder.ex

```diff
@@ -119,6 +131,10 @@ defmodule Zig.Builder do
     end
   end
 
+  defp to_structdef(t = %{cpu: cpu}) do
+    ".{.default_target = .{.cpu_arch = .#{t.arch}, .os_tag = .#{t.os}, .abi = .#{t.abi}, .cpu_model = .{ .explicit = &std.Target.arm.cpu.#{cpu}}}}"
```

Comment:
oh man that's amazing using the .explicit which gets thrown away when it's a non-arm arch.